tests/bluetooth/controller/ctrl_cis_create/src/main.c

```diff
@@ -0,0 +1,468 @@
+
+	/* There should be excactly one host notification */
+	ut_rx_node(NODE_CIS_REQUEST, &ntf, &cis_req);
+	ut_rx_q_is_empty();
+
+	/* Release Ntf */
+	ull_cp_release_ntf(ntf);
+
+	/* Accept request */
+	ull_cp_cc_accept(&conn);
+
+	/* Prepare */
+	event_prepare(&conn);
+
+	/* Tx Queue should have one LL Control PDU */
+	lt_rx(LL_CIS_RSP, &conn, &tx, &local_cis_rsp);
+	lt_rx_q_is_empty(&conn);
+
+	/* Done */
+	event_done(&conn);
+
+	/* Prepare */
+	event_prepare(&conn);
+
+	/* Release Tx */
+	ull_cp_release_tx(&conn, tx);
+
+	/* Rx */
+	lt_tx(LL_CIS_IND, &conn, &remote_cis_ind);
+
+	/* Done */
+	event_done(&conn);
+
+	/* */
+	while (!is_instant_reached(&conn, remote_cis_ind.conn_event_count)) {
+		/* Prepare */
+		event_prepare(&conn);
+
+		/* Tx Queue should NOT have a LL Control PDU */
+		lt_rx_q_is_empty(&conn);
+
+		/* Done */
+		event_done(&conn);
+
+		/* There should NOT be a host notification */
+		ut_rx_q_is_empty();
+	}
+
+	/* Emulate CIS becoming established */
+	/* Prepare */
+	event_prepare(&conn);
+
+	/* Tx Queue should NOT have a LL Control PDU */
+	lt_rx_q_is_empty(&conn);
+
+	/* Done */
+	event_done(&conn);
+
+	/* There should NOT be a host notification */
+	ut_rx_q_is_empty();
+
+	/* Prepare */
+	event_prepare(&conn);
+
+	/* There should be excactly one host notification */
+	ut_rx_node(NODE_CIS_ESTABLISHED, &ntf, &cis_estab);
+	ut_rx_q_is_empty();
+
+	/* Done */
+	event_done(&conn);
+
+	zassert_equal(ctx_buffers_free(), test_ctx_buffers_cnt(),
+		      "Free CTX buffers %d", ctx_buffers_free());
+}
+
+/*
+ * Central-initiated CIS Create procedure.
+ * Central requests CIS, peripheral Host rejects.
+ *
+ * +-----+                    +-------+                    +-----+
+ * | UT  |                    | LL_S  |                    | LT  |
+ * +-----+                    +-------+                    +-----+
+ *    |                           |                           |
+ *    |                           |   LL_CIS_REQ              |
+ *    |                           |<--------------------------|
+ *    |                           |                           |
+ *    |      LE CIS Request       |                           |
+ *    |<--------------------------|                           |
+ *    | LE CIS Request            |                           |
+ *    | Decline                   |                           |
+ *    |-------------------------->|                           |
+ *    |                           |                           |
+ *    |                           | LL_REJECT_EXT_IND         |
+ *    |                           |-------------------------->|
+ *    |                           |                           |
+ */
+static void test_cc_create_periph_rem_host_reject(void)
+{
+	struct node_tx *tx;
+	struct node_rx_pdu *ntf;
+	struct node_rx_conn_iso_req cis_req = {
+		.cig_id = 0x01,
+		.cis_handle = 0x00,
+		.cis_id = 0x02
+	};
+	struct pdu_data_llctrl_reject_ext_ind local_reject = {
+		.error_code = 0x17,
```

Comment:
Is this an HCI error code? If so, please use the macros from `hci_err.h`

Reply:
Here it is just a number that should match between the call to reject and the contents of the PDU

Reply:
But shouldn't it be defined somewhere then, so that it is easier to match?

Reply:
Will do exactly that